template_feed/Microsoft.DotNet.Common.ItemTemplates/content/WebConfig/web.config

```
@@ -4,13 +4,15 @@
   <!-- To customize the asp.net core module uncomment and edit the following section. 
   For more info see https://go.microsoft.com/fwlink/?linkid=838655 -->
   <!--
-  <system.webServer>
-    <handlers>
-      <remove name="aspNetCore"/>
-      <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified"/>
-    </handlers>
-    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" />
-  </system.webServer>
+  <location path="." inheritInChildApplications="false">
+    <system.webServer>
+      <handlers>
+        <remove name="aspNetCore"/>
+        <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified"/>
```

```suggestion
        <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModuleV2" resourceType="Unspecified"/>
```

Is the change needed. However, in 2.1, ANCMV2 isn't present in the hosting bundle (which may change).